deny.toml

```diff
@@ -19,6 +19,20 @@ skip = [
     # up.
     { name = "socket2", version = "0.4.9" },
     { name = "windows-sys", version = "0.42.0" },
+    { name = "windows-sys", version = "0.45.0" },
+    { name = "windows-targets", version = "0.42.0" },
+    { name = "windows_aarch64_gnullvm", version = "0.42.0" },
+    { name = "windows_aarch64_msvc", version = "0.42.0" },
+    { name = "windows_i686_gnu", version = "0.42.0" },
+    { name = "windows_i686_msvc", version = "0.42.0" },
+    { name = "windows_x86_64_gnullvm", version = "0.42.0" },
+    { name = "windows_x86_64_gnu", version = "0.42.0" },
+    { name = "windows_x86_64_msvc", version = "0.42.0" },
+    # Newer versions of crates like `tempfile` are held back by crates like `atty`.
+    # This is very Unfortunate as we don't actually use these platforms.
+    { name = "hermit-abi", version = "0.1.6" },
+    { name = "hermit-abi", version = "0.2.6" },
+    { name = "redox_syscall", version = "0.2.10" },
```

Comment:
Oooof. I'm okay with this if you are willing to be on the hook for wrangling all these duplicate deps over time. (Periodically chasing new versions and sending dependency bump PRs around.)

Reply:
Yeah I can do that!

Reply:
I'm also happy to help as I find myself wrangling many of these duplicates in the cloud repo too :-(